frequently-asked-questions/README.md

```diff
@@ -20,19 +20,19 @@ All tokens have fully vested since August 2023.
 
 ### **What is the Roadmap?**
 
-The roadmap is detailed in the [THORChain Ecosystem Q4`23 Report & 2024 Roadmap](https://medium.com/thorchain/thorchain-ecosystem-q4-23-report-2024-roadmap-1290fde4afa0) article.
+The roadmap is detailed in the [THORChain Q2 2024 Ecosystem Report](https://medium.com/thorchain/thorchain-q2-2024-ecosystem-report-13911bd5c7a5) article.
 
 ### **Where can I buy Rune?**
 
-Directly on THORChain by swapping any supported asset for Native RUNE or exchanges like Binance list Native RUNE. Only Native RUNE is supported on THORChain.
+Directly on THORChain by swapping any supported asset for RUNE or exchanges like Binance list RUNE. Only RUNE is supported on THORChain.
 
 ### **What is the circulating supply of RUNE?**
 
-Approx. 330M with a max of 500M. Full details at [https://ops.ninerealms.com/network](https://ops.ninerealms.com/network) or [hhttps://runescan.io/addresses](https://runescan.io/addresses/) for a breakdown of Native RUNE.
+Approx. 330M with a max of 500M. Full details at [https://ops.ninerealms.com/network](https://ops.ninerealms.com/network) or [hhttps://runescan.io/addresses](https://runescan.io/addresses/) for a breakdown of the RUNE supply.
 
 ### **What is the inflation rate of RUNE?**
 
-The protocol [inflation](https://docs.thorchain.org/network/emission-schedule) is perdominally determined by the size of the reserve. The formula is: `Reserve/EmissionCurve/BlocksPerYear/`, thus gradually decreases every block. In early 2024, with an active reserve of 83m RUNE, the dailyinflation is around 28.5k RUNE. See the  [THORChain Tokenomics article](https://medium.com/thorchain/thorchain-tokenomics-what-is-rune-52d339633260) for more information.
+The protocol [inflation](https://docs.thorchain.org/network/emission-schedule) is perdominally determined by the size of the reserve. The formula is: `Reserve/EmissionCurve/BlocksPerYear/`, thus gradually decreases every block. In early 2024, with an active reserve of 83m RUNE, the dailyinflation is around 28.5k RUNE. See the [THORChain Tokenomics article](https://medium.com/thorchain/thorchain-tokenomics-what-is-rune-52d339633260) for more information.
```

Comment:
predominantly

Reply:
this is fine for now, but going forward let's refrain from having specific values as that just becomes out-dated.